persists/backend_server/server.bal

```diff
@@ -0,0 +1,65 @@
+import ballerina/http;
+import ballerina/persist;
+import ballerina/random;
+
+final Client orderDatabase = check new;
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"]
+    }
+}
+service /sales on new http:Listener(9090) {
+    // Example: http://localhost:9090/sales/orders
+    isolated resource function get orders() returns Order[]|error {
+        return from Order entry in orderDatabase->/orders(targetType = Order)
+            select entry;
+    };
+
+    // Example: http://localhost:9090/sales/orders/HM-238
+    isolated resource function get orders/[string id]() returns Order|http:BadRequest {
```

Comment:
We need to be clear here. If the order with `id` is not there, then it should be `http:NotFound`. It is not correct to send a `BadRequest` here.

Reply:
yes,  I will update

Reply:
updated.